plugins/ipam/static/README.md

```diff
@@ -38,8 +38,17 @@ static IPAM is very simple IPAM plugin that assigns IPv4 and IPv6 addresses stat
 ## Network configuration reference
 
 * `type` (string, required): "static"
-* `addresses` (array, required): an array of arrays of ip address objects:
+* `addresses` (array, optional): an array of arrays of ip address objects:
```

Comment:
That's incorrect even in master - it's an array of ip addresses, not an array of arrays.

Reply:
Fixed.